app/page.tsx

```diff
@@ -6,13 +6,12 @@ import Chat from '@/components/chat';
 import Threads from '@/components/threads';
 import { ChatContextProvider } from '@/contexts/chat';
 import { NavContext } from '@/contexts/nav';
-import { tildy } from '@/config/assistant';
 import ExploreModal from '@/components/explore/ExploreModal';
 import { useDisclosure } from '@nextui-org/react';
 
 function RunFile() {
   const [script, _setScript] = useState<string>(
-    useSearchParams().get('file') ?? tildy
+    useSearchParams().get('file') ?? ''
```

Comment:
Changing this is really all I wanted to do. However, this caused a bunch of errors to start popping up on launch, staring with the `getMe` stuff.

Most of the other changes are resolving the errors that popped up as a result.